src/reference/interactive/index.pug

```diff
@@ -86,62 +88,67 @@ block reference
     .figure
         img(src='/img/reference/interactive/InteractiveHierarchy.svg' alt='Relationship between elements in the interactive hierarchy.')
 
-    p.caption Relationship between elements in the interactive hierarchy.
+    p.caption Diagram showing the relationship between elements in the interactive hierarchy.
 
     h3#scenes Scenes
 
-    p A scene is a named collection of controls. These controls get arranged within the scene on a grid which controls how they are displayed to participants. The #[a(href='#the-gameclient') Game Client] can add or remove controls from a scene and control which participants are seeing which scene throughout the session. Scenes can be used to group controls together in a coherent fashion that has some meaning to the experience.
+    p A scene is a named collection of controls. Within a scene, controls are arranged on a grid. This grid layout determines how they are displayed to participants.
+    
+    p The #[a(href='#the-gameclient') Game Client] can add or remove controls from a scene. It controls which scenes (and controls) are shown to which participants throughout the session. Scenes are used to group controls together in a coherent fashion that is meaningful to the experience.
 
-    p For example, in an adventure game, you might have a "Battle" scene which is displayed when a broadcaster is in battle and a "Field" scene which is displayed when they are walking around in the game world.
+    p For example, in an adventure game, you might have a "Battle" scene displayed when a broadcaster's game character is in battle; a "Field" scene displayed when the character is walking around in the game world.
 
-    p All #[a(href='#an-interactive-project') Interactive Projects] contain a default scene. Without any intervention from the #[a(href='#the-gameclient') Game Client] all participants and #[a(href='#groups') groups] will start on the default scene.
+    p All #[a(href='#an-interactive-project') Interactive Projects] has a default scene. Without any intervention from the #[a(href='#the-gameclient') Game Client] , all participants and #[a(href='#groups') groups] are shown the default scene.
```

Comment:
"All Interactive Projects have a default scene" - 'Projects' is plural, so we should use 'have' instead of 'has'